tests/puppeteer/campaigns/data/faker/product.js

```diff
@@ -0,0 +1,15 @@
+let faker = require('faker');
+
+module.exports = class Product {
+
+  constructor(productType = 'Simple product', productHasCombinations = false, wantedQuantity = '1') {
+    this.name = faker.commerce.productName().toUpperCase();
+    this.description = faker.lorem.sentence();
+    this.reference  = faker.random.alphaNumeric(7);
+    this.quantity  = faker.random.number({'min': 1, 'max': 9}).toString();
+    this.quantity_wanted = wantedQuantity;
+    this.price  = faker.random.number({'min': 10, 'max': 20}).toString();
```

Comment:
Same here, might want to pass this as a parameter:
```suggestion
    this.price  = price || faker.random.number({'min': 10, 'max': 20}).toString();
```

Reply:
Done, thanks